docs/optimization_guide/nncf/ptq/basic_quantization_flow.md

```diff
@@ -57,15 +57,20 @@ The transformation function is a function that takes a sample from the dataset a
        :language: python
        :fragment: [dataset]
 
+If there is no framework dataset object, you can create your own entity that implements the ``Iterable`` interface in Python, for example the list of images, and returns data samples feasible for inference. In this case, a transformation function is not required.
```

Comment:
Hi Helena, I absolutely agree with you that `for example the list of images` is confusing.The main meaning of this sentence that nncf.Dataset supports for any iterable Python object over a dataset and user can implement or use any entity that implements Python iterable interface . If custom or framework dataset returns data samples feasible for inference then a transformation function is not required.

I would recommend to reformulate this sentence:
```
If there is no framework dataset object, you can create your own entity that implements the ``Iterable`` interface in Python. If dataset object returns data samples feasible for inference then a transformation function is not required.
```
I don't seen any reason to add example here because example will include the following code:
```
model_inputs = [...] # model(model_inputs[0])

calibration_dataset = nncf.Dataset(model_inputs)
```
Not sure if this will bring any additional information. If you think otherwise, we can do it.

Reply:
@helena-intel , if you have a suggestion how to better formulate it, feel free to share it here.

Reply:
I think an example of such an iterable would be very useful, also because the PTQ OpenVINO example use an Ultralytics or PyTorch dataloader, and if you're not familiar with them, it may not be obvious what they return, how they handle batching, etc. I'll try to create the most basic NNCF PTQ example today. We can see how to incorporate that in either docs or examples, but that can be in a future PR.